src/tests/resistance/random_bytes.rs

```diff
@@ -0,0 +1,43 @@
+use std::time::Duration;
+
+use tempfile::TempDir;
+use tokio::time::sleep;
+
+use crate::{
+    fuzzing::{random_bytes, seeded_rng},
+    setup::node::{Node, NodeType},
+    tools::synth_node::SyntheticNode,
+};
+
+const ITERATIONS: usize = 20;
+const DISCONNECT_TIMEOUT: Duration = Duration::from_secs(2);
```

Comment:
Uh, does this test lasts for 40 seconds? 😅

Reply:
We should see if we can avoid using `sleep` and use something else instead...

Reply:
Yeah, try to find the lowest timeout value empirically

Reply:
done